Makefile.config.in

```diff
@@ -230,6 +221,9 @@ NAKED_POINTERS=false
 STDLIB_MANPAGES=@build_libraries_manpages@
 WITH_OCAMLDOC=@with_ocamldoc@
 WITH_OCAMLTEST=@ocamltest@
+# This is still processed in configure, but is not needed by the build system
```

Comment:
I fear that this comment might become confusing in the future, it seems better to document the current state rather than the history of this Makefile.

Reply:
Ah, the idea was meant to be a reminder that if/when `SHEBANGSCRIPTS` is removed, it would also be possible to remove `shebangscripts` from `configure.ac`, but I see what you mean. I think it best therefore to remove the comment entirely and trust that whoever removes `SHEBANGSCRIPTS` to consider re-checking for any other uses of `shebangscripts` (and so remove it, if it is truly unused at that point)